### PR TITLE
Cool Bit

### PR DESCRIPTION
**Pull request by HGodal**

### CODE REVIEW

File: src/TrainingEnvironment.py [hunk 1]
```diff
@@ -34,13 +35,21 @@
...
+# Allows the user to specify the max episode step
+# when calling the program
+try:
+    env._max_episode_steps = int(sys.argv[1])
+except (ValueError, IndexError):
+    env._max_episode_steps = 500
+
+print(f"Training agents with max steps = {env._max_episode_steps}:\n")
+
 # Population
```
Comment (bcwein) [inline src/TrainingEnvironment.py:53]: 😎

File: src/functions.py [hunk 2]
```diff
@@ -380,8 +380,7 @@ def breedCrossover(nn1, nn2):
     """
     # Choosing either input -> hidden-layer or hidden-layer -> output
     layer = random.randint(0, 1)
-    # layer = 1
-
+
```
Comment (bcwein) [inline src/functions.py]: Build error here.

**Review by bcwein — CHANGES_REQUESTED** [review]
Line 383 has build error. Minor whitespace adjustment.

Also, add a line in the readme for how to use this feature.

File: src/functions.py (hunk 2 above)
Comment (HGodal) [reply src/functions.py]: 👍🏼

**Review by bcwein — APPROVED** [review]
Good! This can now be merged.

**Review by bcwein — APPROVED** [review]

**Pull request merged**